src/server/app.js

```diff
@@ -54,6 +54,8 @@ app.use('/api/', function (req, res) {
   res.status(404).send("Sorry can't find that!");
 });
 
+// If "/api" is called, redirect to the API documentation.
```

Comment:
Remove comment. This file should not be changed in your PR. Use `git restore src/server/app.js` to reset it.